reloader/post_install.md

```diff
@@ -0,0 +1,138 @@
+## Reloader
```

Comment:
I think that is a long post_install 
can we have a smaller necessary version of this which is required to run and interact with the application ? 

```
A final optional file is the post_install.md which the user will be able to view in the web UI to see additional instructions on how to use the application. It's recommended to keep this short and succinct, if you need to manage LOTS of content, then please create a community learn guide and link to it from within the post_install.md
```

Reply:
ok, i shorted it down. The link for more details is also there :)